.github/workflows/toxic.yml

```diff
@@ -21,11 +21,16 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: process event
+      id: process_event
+      uses: olegtarasov/get-tag@v2.1.2
+      with:
+        tagRegex: "(${{ env.IMAGE_VERSION_PREFIX }}).*"
     - name: dryrun-build
-      if: ${{ github.event_name == 'pull_request' }}
+      if: ${{ github.event_name == 'pull_request' || ( github.event_name == 'release' && startsWith( steps.process_event.outputs.tag, env.IMAGE_VERSION_PREFIX ) && github.event.release.prerelease ) }}
```

Comment:
same question here, what's the difference between a pre-release and the pull_request?

Reply:
See answer above.